Travel_fellowships.md

```diff
@@ -92,6 +92,13 @@ However, this fellowship may cover the rental cost of the large hardware (for ex
 The cost will be capped to a maximum of $40 (USD) each for small hardware such as headphones, webcam, speaker, and/or microphone.
 If the cap is insufficient, applicants can describe their hardware needs in their application.
 
+### Expenses and Cancellation
+
+We recommend you purchase travel insurance, and include this as part of your expenses claim.
+In the event of cancellation, we would expect to reimburse your travel insurance premium and any insurance excess fee (as long as the total reimbursement does not exceed the originally agreed award).
+
+If the awardee elects not to purchase travel insurance, any reimbursement is at the discretion of the board.
```

Comment:
Just noting that this document switches back and forth between 2nd person ("you"), third person singular ("the awardee"), and third person plural ("applicants"). Probably not worth fixing, though?

Reply:
> Just noting that this document switches back and forth between 2nd person ("you"), third person singular ("the awardee"), and third person plural ("applicants"). Probably not worth fixing, though?

Yeah I think that's worth maybe revising, maybe in a followup pull PR?